configuration/uart.md

```diff
@@ -57,7 +55,7 @@ There are several ways to configure the UART hardware.
 
 #### config.txt
 
-Adding `enable-uart=1` to `/boot/config.txt` should be enough to enable the first two UART ports.
+On most devices, adding `enable-uart=1` to `/boot/config.txt` should be enough to enable the first two UART ports. On compute modules, you must also add a device-tree overlay eo enable the particular UART you want to use.
```

Comment:
I'm not sure if "enable the first two UART ports" is actually what this option does? IIRC it's for ensuring the UART on the GPIO header is usable?

Reply:
I think you're right. Is the source code available to have a look and know for sure?

Reply:
`config.txt` is handled by `start.elf`, and that's closed source.